scripts/update_assets/README.md

```diff
@@ -0,0 +1,62 @@
+# Updating assets 
+
+## Description 
+The motive is to make it easier for the customer to update their existing assets. There are two codes in the project: 
+* get_assets.py; gets the list of existing assets and creates a csv file named assets.csv in the directory.
+* update_assets.py: takes the modified CSV file and send a request for update. 
+
```

Comment:
I think you can remove those empty lines. One line between paragraphs is enough.

Reply:
Done!